draft-ietf-mls-protocol.md

```diff
@@ -118,27 +118,27 @@ Instructions are on that page as well. Editorial changes can be
 managed in GitHub, but any substantive change should be discussed on
 the MLS mailing list.
 
-A group of agents who want to send each other encrypted messages needs
+Users who want to send each other encrypted messages needs
```

Comment:
Subject/verb agreement "Users ... needs".  Re-introducing "A group of..." would fix this.

Reply:
Fixed.